### PR TITLE
Update scriptType description

### PR DESCRIPTION
**Pull request by sonic1981**
It's not very clear what exactly the various options are for the scriptType input of this command. I've tried to make this more clear and to highlight the actual variable options and when they are valid.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@sonic1981 : Thanks for your contribution! The author(s) have been notified to review your proposed change. @UshaN

**Comment by PRMerger12** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change. @UshaN

**Comment by PRMerger12** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/tasks/deploy/azure-cli.md [hunk 1]
```diff
@@ -51,7 +51,18 @@ cross-platform agents running on Linux, macOS, or Windows operating systems.
 </tr>
 <tr>
     <td><code>scriptType</code><br/>Script Type</td>
-    <td>(Required) Type of script: <b>PowerShell/PowerShell Core/Bat/Shell</b> script. Select <b>bash/pscore</b> script when running on Linux agent or <b>batch/ps/pscore</b> script when running on Windows agent. PowerShell Core script can run on cross-platform agents (Linux, macOS, or Windows)</td>
+  <td>(Required) Type of script: <b>PowerShell/PowerShell Core/Bat/Shell</b> script.</br>When running on a <b>Linux agent</b> select on of the following:</br>
```
Comment (juliakm) [inline docs/pipelines/tasks/deploy/azure-cli.md:54]:
```suggestion
  <td>(Required) Type of script: <b>PowerShell/PowerShell Core/Bat/Shell</b> script.</br>When running on a <b>Linux agent</b> select one of the following:</br>
```

**Review by juliakm — COMMENTED** [review]
Could you please fix the small typo?

**Comment by PRMerger20** [comment]
@sonic1981 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by sonic1981** [comment]
Is there a reason this hasn't been merged in yet?

**Comment by ktoliver** [comment]
> Is there a reason this hasn't been merged in yet?

We're waiting for the author @juliakm to sign off.

**Comment by ktoliver** [comment]
image

**Comment by ktoliver** [comment]
@juliakm @N-Usha Could you please confirm that the update is good to merge? Thanks!

**Comment by sonic1981** [comment]
So it's becoming perfectly evident that this is getting ignored. I've had no feedback and the merge is blocked for unknown reasons. If I nothing happens soon I'm just going to close this PR and give up trying to improve **your** documentation. @N-Usha

**Comment by Jak-MS** [comment]
@juliakm @steved0x @N-Usha - can you take a look and sign off if this is good to merge? thanks.

**Comment by ktoliver** [comment]
@KathrynEE Would you be able to see if someone who works on pipelines content can review this community contribution? Thank you.

**Comment by sonic1981** [comment]
Well this is turning into an absolute shambles...

**Comment by KathrynEE** [comment]
@sonic1981 s- Thank you for the contribution and apologies for the delay. The article should be updated later today.

**Pull request merged**